libsession/src/main/java/org/session/libsession/snode/SnodeAPI.kt

```diff
@@ -57,7 +57,7 @@ object SnodeAPI {
      */
     internal var clockOffset = 0L
 
-    val nowWithOffset
+    public val nowWithOffset
```

Comment:
I hate to be annoying but if we add the `@JvmField` does it let us access it to same way without the `.INSTANCE` references throughout the Java side?

Reply:
All good - so `@JvmField` complains because it doesn't have a backing value (`This annotation is not applicable to target member property without backing field or delegate`) but `@JvmStatic` seems to work

Reply:
ahh yeah makes sense, I guess `@JvmStatic` makes it work because it's just the a get function with nice syntax